CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Fixed
+- Changing font size now also takes an effect in CEA-608 captions with cap from 50% to 200%. Larger than default font size also disables the grid view.
```

Comment:
Nit, but was that actually a bug? If not, I'd add it to the `Changed` or `Added` section. If we keep it in `Fixed`, we might want to rephrase, e.g.:

```suggestion
- font size adjustments for CEA-608 captions not taking effect.
```

Reply:
I mean its considered as bug but we can also put it as Added as its new behaviour.